blob/src/main/java/io/crate/blob/v2/BlobIndicesService.java

```diff
@@ -58,7 +58,10 @@
         "index.blobs.enabled", false, Setting.Property.IndexScope);
     public static final Setting<String> SETTING_INDEX_BLOBS_PATH = Setting.simpleString(
         "index.blobs.path", Setting.Property.IndexScope);
-    public static final Setting<String> SETTING_BLOBS_PATH = Setting.simpleString("blobs.path", Setting.Property.NodeScope);
+    public static final Setting<String> SETTING_BLOBS_PATH = Setting.simpleString(
+        "blobs.path", Setting.Property.NodeScope);
+    public static final Setting<Boolean> SETTING_HTTP_COMPRESSION = Setting.boolSetting(
```

Comment:
there is already a definition for it in `HttpTransportSettings`, I'd use that instead and instead of calling `getDefault` set it to false